adkintunmobile-server/populate/Dockerfile

```diff
@@ -0,0 +1,51 @@
+## Header
+FROM python:3.5.1-slim
+MAINTAINER Boris Romero <boris@niclabs.cl>
+
+
+## Dependencies
+# install apt dependencies
+RUN apt-get update && \
+	apt-get install -y \
+		gcc \
+		git \
+		libpq-dev \
+		make \
+		wget
+
+# clone project
+RUN mkdir /adk && \
+	cd /adk && \
+	git clone https://github.com/niclabs/AdkintunMobile-Server.git && \
+	cd AdkintunMobile-Server && \
+	git checkout 451e2637dd7bf15d0574b49d2397e90f548ef6ab 
```

Comment:
I spoke with @franchoco about this once... He said that is a good idea because when you want a new build you can reuse the CACHE if you just change the checkout to a newer commit. So docker realizes is different from this point and reuse the previous instructions....
We came to an agreement to do a checkout against a tag instead of a commit.
Check https://github.com/niclabs/docker/blob/master/ratadns/fievel/Dockerfile